Dockerfile-ubuntu.template

```diff
@@ -288,6 +288,8 @@ VOLUME $RABBITMQ_DATA_DIR
 # https://docs.docker.com/samples/library/ubuntu/#locales
 ENV LANG=C.UTF-8 LANGUAGE=C.UTF-8 LC_ALL=C.UTF-8
 
+COPY rabbitmq.conf /etc/rabbitmq/rabbitmq.conf
```

Comment:
I still think we should probably put this in `conf.d` instead, but I don't feel strongly enough about it to fight too hard.

If we keep it like this, I'd nit: `COPY rabbitmq.conf /etc/rabbitmq/` (dropping the RHS filename)

Reply:
`/etc/rabbitmq` is the right place for the `rabbitmq.conf` unless you suggest we use a config file fragment.

Reply:
I've dropped the rhs filename.

Reply:
I have actually now switched this to fragments as well, as there seems to be slightly more consensus around that approach.